cmd/ocitools/generate.go

```diff
@@ -184,16 +184,18 @@ func setupSpec(g generate.Generator, context *cli.Context) error {
 		}
 	}
 
+	hostSpecific := context.GlobalBool("host-specific")
+
 	privileged := false
 	if context.IsSet("privileged") {
 		privileged = context.Bool("privileged")
 	}
-	g.SetupPrivileged(privileged)
+	g.SetupPrivileged(privileged, hostSpecific)
```

Comment:
Thinking about this more, I think that we shouldn't pass this as part of the API. We should have a call to enable hostSpecific on the generate object and then use that variable in individual APIs

```go
g := generate.New()
g.EnableHostSpecific()
g.SetupPrivileged()
...
g.SaveToFile()
```